http-clients/src/main/java/com/palantir/remoting3/clients/UserAgent.java

```diff
@@ -1,104 +0,0 @@
-/*
- * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.remoting3.clients;
-
-import static com.google.common.base.Preconditions.checkArgument;
-
-import java.util.List;
-import java.util.Optional;
-import org.immutables.value.Value;
-
-/**
- * Constructs, validates, and formats a canonical User-Agent header. Because the http header spec
- * (https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2) requires headers to be joined on commas, individual
- * {@link Agent} header strings must never contain commas.
- */
-@Value.Immutable
-@ImmutablesStyle
-public interface UserAgent {
```

Comment:
:+1: moved to http-remoting-api: https://github.com/palantir/http-remoting-api/pull/100/files#diff-04661908a4686a62ad7b9699fbd20e76R32